package.json

```diff
@@ -214,6 +214,9 @@
       "!**/{.DS_Store,.git,.hg,.svn,CVS,RCS,SCCS,__pycache__,thumbs.db,.gitignore,.gitattributes,.editorconfig,.flowconfig,.yarn-metadata.json,.idea,appveyor.yml,.travis.yml,circle.yml,npm-debug.log,.nyc_output,yarn.lock,.yarn-integrity}",
       "node_modules/spellchecker/build/Release/*.node",
       "node_modules/websocket/build/Release/*.node"
-    ]
+    ],
+    "engines": {
```

Comment:
Looks like this is causing CI failures, since it's under the `build` key. Maybe pull it higher in the file, before the `build` key. I kind of like having all that stuff at the bottom of the file, anyway.

Reply:
Good catch, didn’t mean to include it under `build`.